app/services/queue_listener.rb

```diff
@@ -40,13 +41,20 @@ def perform(sqs_message, params)
       when UPDATE_MEMBER
         update_member(params)
 
+      when SUBSCRIPTION_CANCELLATION
+        cancel_subscription(params[:params])
+
       else
         raise ArgumentError, "Unsupported message type: #{params[:type]}"
     end
   end
 
   private
 
+  def cancel_subscription(params)
+    client.cancel_subscription(params)
```

Comment:
Hey @Tuuleh, I know I'm late to the party but this is the exact kind of case where I think we should check for errors. If the call to AK's api fails for some reason this will silently fail, and we would never find out about it.